frontend/public/components/utils/dropdown.jsx

```diff
@@ -293,7 +293,7 @@ export class Dropdown extends DropdownMixin {
       ? <div onClick={this.toggle} className="dropdown__not-btn" id={this.props.id}>
         <span className="dropdown__not-btn__title">{title}</span>&nbsp;<Caret />
       </div>
-      : <button onClick={this.toggle} type="button" className="btn btn-default btn--dropdown dropdown-toggle" id={this.props.id}>
+      : <button onClick={this.toggle} type="button" className={`btn btn-default btn--dropdown dropdown-toggle ${className}`} id={this.props.id}>
```

Comment:
I think it's better to have separate props for button className and dropdown className. It's confusing that it's used in different ways depending on `props.noButton`